sentry-sidekiq/spec/sentry/sidekiq/util_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'spec_helper'
+
+RSpec.describe Sentry::Sidekiq::Util do
+  describe '.transaction_name_from_context' do
+    subject(:transaction_name) { described_class.transaction_name_from_context(context) }
+
+    context 'when the context is a job' do
+      let(:context) { { 'class' => 'FooJob' } }
+
+      it 'extracts the class' do
+        expect(transaction_name).to eq('Sidekiq/FooJob')
+      end
+    end
+
+    context 'when the context is a wrapped job' do
+      let(:context) { { 'wrapped' => 'FooJob', 'class' => 'WrapperJob' } }
+
+      it 'extracts the class' do
```

Comment:
it'd be better if the case description can be more clear. like `"extracts the wrapped job's name"`.